app/views/groups/_mentor_group.html.erb

```diff
@@ -24,20 +28,7 @@
       <% end %>
     </div>
 
-  <div class="flex" style="display: flex; flex-direction: row; align-items: center;">
-    <div class="row">
-      <div class="mentor-cta">
-        <div class="btn">
-          <%= link_to "members", users_group_path %>
-        </div>
-      </div>
-        <div class="btn">
```

Comment:
Could the "leave mentor program" please be put back in? Since there is no other way for people to look for a new mentor if they're unsatisfied, or for mentors to leave the program, as we haven't put in a dashboard and approve/reject method, having an option to leave is essential. Thanks